uber/templates/regform.html

```diff
@@ -92,7 +92,6 @@
     $(function () {
         staffingClicked();
         if ($.field('staffing')) {
-            {% if attendee.shifts %} $.field('staffing').prop('readonly', true).prop('title', 'Please {{ "contact your department head to drop shifts" if c.AT_THE_CON else "unassign yourself from shifts" }} before changing your volunteer status.'); {% endif %}
```

Comment:
This is interesting, a detail I knew in the deep recesses of my mind, but completely forgot about. Readonly checkboxes aren't actually readonly, you can still click and change them because you aren't changing their `value`, only their `checked` state.

You can make a checkbox actually readonly by setting its `disabled` property, but then it won't be posted to the server (this is the bug that was deleting everyone's shifts).

Instead of doing this in javascript, I think the correct solution is to render a disabled checkbox & a hidden input that will actually post the value.